require('dotenv').config();

const Task = require('../models/Task');
const Sedes = require('../models/Sedes');

const getTasks = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const getTask = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const addTask = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const updateTask = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const deleteTask = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const getSedes = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const getSede = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const addSede = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const updateSede = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

const deleteSede = async (req, res) => {
    try {
        
    } catch (error) {
        res.status(500).json({
            typeError: "",
            message: "",
            data: {},
            error: error
        });
    }
}

module.exports = {
    
}